{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "6ee6a822_ee42f1bb",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1163549
      },
      "writtenOn": "2024-03-25T23:58:37Z",
      "side": 1,
      "message": "PTAL\n\nLooks like there weren\u0027t any new failures as a result of the OS/driver upgrade.",
      "revId": "88a80ef3fbdb96e6d28032d66ef949931137f456",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "ad88364a_b7157517",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1001921
      },
      "writtenOn": "2024-03-26T13:29:00Z",
      "side": 1,
      "message": "Cool, no failures!\nI guess 31.0.101.2114 isn\u0027t that different from 31.0.101.2111.",
      "revId": "88a80ef3fbdb96e6d28032d66ef949931137f456",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}